pkg/project/file.go

```diff
@@ -27,11 +28,12 @@ func (f File) Detect() (Result, bool, error) {
 
 	lines, err := readFile(fp, 2)
 	if err != nil {
-		return Result{}, false,
-			Err(fmt.Sprintf("error reading file: %s", err))
+		return Result{}, false, Err(fmt.Sprintf("error reading file: %s", err))
 	}
 
-	result := Result{}
+	result := Result{
+		Folder: filepath.Join(fp, ".."),
```

Comment:
It's clearer to use `Folder: filepath.Dir(fp),` unless that could cause problems from sometimes returning `"."` instead of empty string `""`?

Reply:
makes more sense to `Dir(..)`, will change in the next fixup.